homepage/download.html

```diff
@@ -54,11 +54,13 @@ <h2>A thermodynamic and geophysics toolkit for the Earth and planetary sciences<
 
 <h3>Download:</h3>
 
+The links below are for the latest released version of BurnMan. You may also wish to check out the latest development version, which you can access by clicking the "Source" link above.
 <p>
 <ul>
-<li><a href="https://github.com/geodynamics/burnman/releases/tag/v0.9.0">Latest release:</a> <a href="https://github.com/geodynamics/burnman/releases/download/v0.9.0/burnman-0.9.0.zip">burnman-0.9.0.zip</a> </li>
-<li><a href="doc-0.9.0/">Online Documentation</a></li>
-<li><a href="https://github.com/geodynamics/burnman/releases/download/v0.9.0/BurnMan-0.9.0.pdf">PDF Manual</a></li>
+<li><a href="https://github.com/geodynamics/burnman/releases/tag/v0.10.0">Latest release:</a>
+    <a href="https://github.com/geodynamics/burnman/releases/download/v0.10.0/burnman-0.10.0.zip">burnman-0.10.0.zip</a> </li>
```

Comment:
This files does not exist, yet, only https://github.com/geodynamics/burnman/archive/refs/tags/v0.10.0.zip gets generated automatically. I don't like the filenames, so I would download, rename. and reupload the files to the release.

Reply:
Thanks! Done.